e2e/test-page.html

```diff
@@ -171,19 +172,15 @@ <h3>Embedded styles</h3>
         .b {
           fill: #020a1a;
         }
-
         .c {
           fill: #53cf92;
         }
-
         .d {
           fill: #57b2f8;
         }
-
         .e {
           fill: #fec83c;
         }
-
```

Comment:
ah, that's how you modified the SVG, it should be all good then 👌

Reply:
Perfect, thanks.